app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     defaultConfig {
         applicationId "covid.trace.morocco"
         resValue "string", "build_config_package", "covid.trace.morocco"
-        minSdkVersion 22
+        minSdkVersion 21
```

Comment:
Not related to your PR. Changing minSdkVersion must be well discussed.

Reply:
this modification is linked to the PR #1, I did it because the BLE is available from version 21.

Reply:
I agree not to be linked to this PR. I corrected it